<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SalomeToolCreateMeshDialog</class>
 <widget class="QDialog" name="SalomeToolCreateMeshDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>420</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Wizard: Creating the Mesh </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="title">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Define the Salome Options to create the mesh</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="meshFileNameLineEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="meshFileNameLabel">
       <property name="text">
        <string>Mesh file Name</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="createMeshButton">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>100</height>
      </size>
     </property>
     <property name="text">
      <string>Create the Mesh</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="consoleTextEdit"/>
   </item>
   <item alignment="Qt::AlignRight">
    <widget class="QPushButton" name="finishButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="text">
      <string>Finish</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
